app/uk/gov/hmrc/cgtpropertydisposals/models/LocalDateUtils.scala

```diff
@@ -21,6 +21,41 @@ import java.time.LocalDate
 import cats.Order
 
 object LocalDateUtils {
+  val monthNames: Map[Int, String] = Map(
+    1  -> "January",
```

Comment:
we should read these from a messages file rather than have the values hard coded here. When we come to do a welsh translation of the service we will need all the strings we need translating inside a single file

Reply:
Done.